.github/maintainers_guide.md

```diff
@@ -3,102 +3,146 @@
 > ⚠️ This maintainers guide is out of date. Progress on updating it is being [tracked](https://github.com/slackapi/node-slack-sdk/issues/748).
 
 This document describes tools, tasks and workflow that one needs to be familiar with in order to effectively maintain
-this project. If you use this package within your own software as is but don't plan on modifying it, this guide is
+this project. If you use this package within your own software but don't plan on modifying it, this guide is
 **not** for you.
 
 ## Tools
 
-All you need to work with this project is a supported version of [Node.js](https://nodejs.org/en/)
-(see `package.json` field "engines") and npm (which is distributed with Node.js).
+Maintaining this project requires installing [Node.js](https://nodejs.org) in your development environment. All of the
+remaining tools are downloaded as `devDependencies`, which means you'll have them available once you run `npm install`
+in a working copy of this repository.
+
+In particular, [`lerna`](https://lerna.js.org/) is a tool you should become familiar with. This project is home to
+several packages, and lerna makes it easier to manage operations across the set of packages. The very next thing you
+should do is run `npx lerna bootstrap` in the root of this project to install dependencies for each of the packages,
+and to allow lerna to link them to each other.
 
 ## Tasks
 
 ### Testing
 
-This package has unit tests for most modules (files) in the `test` directory. The structure mirrors the `lib` directory closely. You can run the entire test suite using the npm script `npm test`. This command is also executed by Travis, the continuous integration service, for every Pull Request and branch. The coverage is computed with the `istanbul` package. The tests themselves are run using the `mocha` test runner.
+This package has tests for individual packages in the respective `test` directory inside those packages. It also has
```

Comment:
First sentence is a little confusing. Suggestion:

This project has tests for individual packages inside of each's respective `test` directory.